index.bs

```diff
@@ -324,6 +324,11 @@ The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "S
     that this does not constitute [=user verification=] because [=TUP=], by definition, is not capable of [=biometric
     recognition=], nor does it involve the presentation of a shared secret such as a password or PIN.
 
+: <dfn>Resident Key</dfn>
+: <dfn>RK</dfn>
+:: A [=resident key=] is a [=credential private keys=] that can be found and used by the authenticator based on only the 
```

Comment:
Something in https://w3c.github.io/webauthn/#op-get-assertion is missing a mention of the fact that keys might only be found if their credential IDs are passed in allowList. Without that, the whole idea of resident vs non-resident keys doesn't really make sense.